docs/conf.py

```diff
@@ -7,6 +7,8 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = "YScope"
+# NOTE: We don't include a period after "Inc" since the theme adds one already.
+copyright = "2023-2024 YScope Inc"
```

Comment:
`Shadows built-in name 'copyright' - Module docs.conf` 
References:
1. https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-project_copyright
2. https://github.com/sphinx-doc/sphinx/issues/8132

```suggestion
project_copyright = "2023-2024 YScope Inc"
```